site/en/getstarted/cluster/install_cluster-ansible.md

```diff
@@ -13,7 +13,7 @@ summary: Learn how to install Milvus cluster with Ansible Controller.
 
 {{tab}}
 
-This topic introduces how to deploy a Milvus cluster. We provide the Docker Host for creating an Ansible playbook and the Container for running the Milvus cluster. The script in this topic is only for running on Ubuntu 20.04 LTS systems. Modify the script commands and parameters if you want to run the Milvus cluster on other versions of operating systems.
+This topic introduces how to deploy a Milvus cluster with Ansible. We provide the Docker Host for creating an Ansible playbook and the Container for running the Milvus cluster. The script in this topic is only for running on Ubuntu 20.04 LTS systems. Modify the script commands and parameters if you want to run the Milvus cluster on other versions of operating systems.
 
 ## Prerequisites
```

Comment:
So the [environment checklist](https://milvus.io/docs/v2.0.x/prerequisite-docker.md) is not applicable to this topic?